hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -27,8 +27,13 @@ public static HawkBuilder init(Context context) {
     if (context == null) {
       throw new NullPointerException("Context should not be null");
     }
-    hawkBuilder = new HawkBuilder(context);
-    return hawkBuilder;
+
+    internal = null;
```

Comment:
we can remove internal= null statement, as default it's null, or you did for something else? cannot find any reason